src/dummy/toga_dummy/widgets/base.py

```diff
@@ -24,7 +24,7 @@ def container(self, container):
         self._set_value('container', container)
 
     def set_enabled(self, value):
-        self._action('set enabled', value=value)
```

Comment:
This one should be reverted; it's just logging what the invocation was, not actually doing anything with the value.

Reply:
Done.  Also fixed up the unites to match.